src/autosavefile.cpp

```diff
@@ -27,6 +27,8 @@
 static const QLatin1String subdir("/autosave");
 static const QLatin1String extension(".mlt");
 
+QString AutoSavePath = NULL;
```

Comment:
Global should static, start with g_ prefix, and initializing QString will NULL is non-sensel as QString is not a pointer.